### PR TITLE
Nodejs week1

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

**Review by dpfernandes — CHANGES_REQUESTED** [review]
@andrii1 you are changing 2.384 files and nobody is going to review it :) 

Please fix this PR(or create a new one) so that you commit only the files that you worked on for the NodeJS-Week1

**Comment by andrii1** [comment]
> @andrii1 you are changing 2.384 files and nobody is going to review it :)
> 
> Please fix this PR(or create a new one) so that you commit only the files that you worked on for the NodeJS-Week1

ok done!

**Review by Divyajg — APPROVED** [review]
Logic flow is impressive.
Great work. 👍

File: nodejs/week1/nodejs-week1/app.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  res.send({ rows });
+});
+
+app.get("/gmail-users", async (req, res) => {
+  const dbResult = await knex.raw(
+    "SELECT * from users WHERE email LIKE '%gmail%'"
+  );
+  const rows = dbResult[0];
+  res.send({ rows });
+});
```
Comment (27ManishaPatel) [inline nodejs/week1/nodejs-week1/app.js]: for gmail user isn’t it LIKE '%@gmail %'?

File: nodejs/week1/nodejs-week1/app.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    "SELECT * from users ORDER by created_at LIMIT 1"
+  );
+  const rows = dbResult[0];
+  if (rows.length === 0) {
+    res.status(404).send("Not found");
+  } else {
+    res.send({ rows });
+  }
+});
+
```
Comment (27ManishaPatel) [inline nodejs/week1/nodejs-week1/app.js]: Good job !

File: nodejs/week1/avg.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  console.log("myArgs: ", arrayNumbers);
+  const average = arrayNumbers.reduce((a, b) => a + b, 0) / arrayNumbers.length;
+  console.log("average: ", average);
+}
+//function to check if numbers
+function onlyNumbers(array) {
+  return array.every((element) => {
+    return !isNaN(element);
+  });
+}
```
Comment (27ManishaPatel) [inline nodejs/week1/avg.js:20]: Great !

**Review by 27ManishaPatel — APPROVED** [review]
Good work !

**Pull request merged**